docusaurus/docs/migration-guide/_category_.json

```diff
@@ -0,0 +1,6 @@
+{
+    "position": 5,
+    "label": "Migration guide",
```

Comment:
```suggestion
    "label": "Migration guides",
```

Reply:
nit - but if this is a collection of different guides, shouldn't this be plural?

Reply:
Fixed this issue and refactored pages to unite all the migration guides under one umbrella.